util/dashboard_config.py

```diff
@@ -398,7 +398,11 @@ def prepare_ontologies(ontologies, ontology_dir, dashboard_dir, make_parameters,
         ont_results_path = os.path.join(ont_dashboard_dir, "dashboard.yml")
         ont_results = ontologies_results[o]
 
-        if os.path.exists(ont_results_path) and config.is_skip_existing():
+        if config.is_skip_existing():
```

Comment:
Why remove the `os.path.exists(ont_results_path)` - what happens if you set "skip existing" to true, and there is no result being generated?

Reply:
Also, maybe add a logging here as well like you do below.

Reply:
I split and added the next condition to check if the ontology was changed and if there was a result; that is more important.

Reply:
Yes but what will happen if you want to skip and there is no result? Then the processing will be skipped without a result file and a way to even get to the second if statement?

Reply:
It keeps without the result. What is the reason to set skip true and still expect to have something?

Reply:
The whole purpose of "is_skip_existing" is that "if a result exists, and the setting is set to skip, skip it" -> this is mostly for developing scenarios where you have to frequently re-run the dashboard

Reply:
I did that in this PR; check if the ontology changed or not because if the result exists, it can still have a failure, so it's skipped without rerunning.

Reply:
I leave it to you - I still find it a bit wrong from the way I think of it, but I trust your judgement and my neocortical availibility is fairly limited. Go ahead!

Reply:
I double-checked and there was a bug in the other condition that was not nested in the condition where it checks the `dashboard.yml` existence.